File: search-service/src/main/java/com/stackroute/eplay/search/domain/MovieEvent.java
package com.stackroute.eplay.search.domain;

import java.util.Date;
import java.util.List;

import org.springframework.data.annotation.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class MovieEvent {
	private int movieEventId; 
	private int movieId; 
	private int theatreId; 
	private int showCount; 
	private Date bookingStartDate; 
	private Date bookingEndDate; 
	private List<Show> shows; 
	private String city;

}

File: Front-end/src/app/movie-event.ts
import { Show } from "./Show";
export class MovieEvent {
    movieEventId: number;
    movieId:number;
    theatreId: number;
    theatreName:string;
    showCount: number;
    bookingStartDate: Date;
    bookingEndDate: Date;
    weeks:number;
    shows:Show[];
    city:string;
    userName:string;
    timings:string;
  }